### PR TITLE
Update querying.ipynb

### PR DESCRIPTION
**Pull request by superstar54**
Fix a typo: `days=12` instead of `days=10000`.

### CODE REVIEW

**Comment by mbercx** [comment]
@unkcpz could you double check this, since you're running the session on Wednesday? 🙃

**Comment by unkcpz** [comment]
Sure @mbercx. Thanks @superstar54, nice catch.

I think the issue is the notebook cell is not compatible with the description, I think other than changing the day older than the 12 days, we need to change the description to 10000 or exactly the number of days it was created. Otherwise, I think no nodes will return.

**Comment by unkcpz** [comment]
As discussed offline, I suggest adding a note that if no new structure data was created by student during the tutorial, this may show an empty list.

File: docs/sections/managing_data/querying.ipynb [hunk 1]
```diff
@@ -2306,7 +2306,7 @@
    "name": "python",
    "nbconvert_exporter": "python",
    "pygments_lexer": "ipython3",
-   "version": "3.9.5"
+   "version": "2.7.18"
```
Comment (unkcpz) [inline docs/sections/managing_data/querying.ipynb]:
```suggestion
   "version": "3.9.5"
```

File: docs/sections/managing_data/querying.ipynb [hunk 2]
```diff
@@ -1095,7 +1095,7 @@
     "\n",
     "You may have multiple criteria that you want to filter for, in which case you can use the logical `or` and `and` operators.\n",
     "Let's say, for example, you want the `QueryBuilder` to retrieve all the crystal structure nodes (`StructureData`) that were created no longer than 12 days ago **and** have an `a` in their UUID.\n",
-    "You can express this criterion by making use of the `and` operator, which allows you to specify multiple filters that all have to be satisfied."
+    "You can express this criterion by making use of the `and` operator, which allows you to specify multiple filters that all have to be satisfied. The query would show an empty list if no new structure data were created within 12 days with `a` in their UUID."
```
Comment (unkcpz) [inline docs/sections/managing_data/querying.ipynb]:
```suggestion
    "You can express this criterion by making use of the `and` operator, which allows you to specify multiple filters that all have to be satisfied.\n",
    "The query would show an empty list if no new structure data were created within 12 days with an `a` in their UUID.\n"
```

**Review by unkcpz — CHANGES_REQUESTED** [review]
@superstar54 thanks! Just minor request.

**Review by unkcpz — APPROVED** [review]
Looks all good! Cheers. 🚀

**Comment by superstar54** [comment]
Thanks for the suggestions. It has been updated.

**Pull request merged**